client/src/com/mirth/connect/client/ui/actions/PasteAction.java

```diff
@@ -35,8 +36,14 @@ public void actionPerformed(ActionEvent e) {
     public boolean isEnabled() {
         if (comp.isVisible() && comp.isEditable() && comp.isEnabled()) {
             try {
-                Transferable contents = Toolkit.getDefaultToolkit().getSystemClipboard().getContents(this);
-                return contents.isDataFlavorSupported(DataFlavor.stringFlavor);
+                Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+                if(null != clipboard) {

```

Comment:
Is this a Yoda Condition? https://en.wikipedia.org/wiki/Yoda_conditions

This is a style issue

Reply:
@ChristopherSchultz Yeah, can we reverse the placement of the ```null``` and the ```clipboard``` variable in the condition? I think that'll make it more consistent with the rest of the conditions throughout Connect. So it'll look like the following instead:
```if (clipboard != null) {```